source/extensions/filters/listener/tls_inspector/tls_inspector.cc

```diff
@@ -330,6 +345,259 @@ void writeEllipticCurvePointFormats(const SSL_CLIENT_HELLO* ssl_client_hello,
   }
 }
 
+// Maps a TLS version uint16_t to its string representation.
+absl::string_view mapVersionToString(uint16_t version) {
```

Comment:
nit: I would have slight preference to moving functions for JA hash computation into a separate file for readability.